guides/hack/01-getting-started/03-starting-a-real-project.md

```diff
@@ -1,7 +1,7 @@
 ## Starting A Real Project
 
 Real projects generally aren't a single file in isolation; they tend to have
-dependencies such as the [Hack Standard Library], and various optional tools.
+dependencies such as the [TypeAssert](https://github.com/hhvm/type-assert), and various optional tools.
```

Comment:
- 'the TypeAssert library' or 'TypeAssert'
- How about 'such as libraries, frameworks, or various optional tools' ? TypeAssert:
  - is a 'necessary evil'
  - it's not clear why it's useful to someone who doesn't yet understand how hack's type system (and runtime enforcement) works
 
 There's things like 'hacktest' that might be a better example, but I don't think one's really needed, and putting something here feels like 'most projects will want this' that I don't realyl want to imply

Reply:
I'll be touching up this file anyway #1127. Changing this link was a knee jerk reaction on hhvm/hsl#168. After rereading this page, I realized that it could use a lot of little touch ups. Closing for now. Thanks for the insights.